plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -545,12 +567,15 @@ static int prepare_dns_reply(u_char *data, const char *name, int type, int *dns_
          len = 12 + IP_ADDR_LEN;
          SAFE_CALLOC(answer, len, sizeof(u_char));
          p = answer;
-      
+
+         /* make TTL big-endian to work with DNS */
+         ttl_bige = htonl(ttl);
```

Comment:
Ah yes you are right. Then the second variable is necessary (even though I don't know the word "bige"). memcpy requires pointers so that a direct call of htonl within memcpy cannot do that.